components/measurement/nettests/WebConnectivity.js

```diff
@@ -369,20 +365,6 @@ const WebConnectivityDetails = ({
         }}
       />
     )
-  } else {
-    // TODO: Remove this block when the first block in this chain is enabled.
-    status = 'error'
-    summaryText = (
-      <FormattedMessage
-        id='Measurement.SummaryText.Websites.Failed'
-        values={{
-          date: date,
-          WebsiteURL: input,
-          network: probe_asn,
-          country: country,
-        }}
-      />
-    )
```

Comment:
This fallback condition might have to remain. I have sought confirmation in the linked issue #426.

Reply:
I have addressed this by restoring this chunk.